include/OpenSpaceToolkit/Astrodynamics/EventCondition.hpp

```diff
@@ -90,6 +90,22 @@ class EventCondition
 
     virtual bool isSatisfied(const Real& currentValue, const Real& previousValue) const;
 
+    /// @brief              Check if the Event Condition is satisfied based on current value and previous value
```

Comment:
```suggestion
    /// @brief              Check if the Event Condition is satisfied based on current state and previous state.
```
Maybe add context on why time is needed here?